### DISCUSSION TITLE
请问各位大佬，过CDN的协议中，有没有比grpc更快，又不被GFW针对封禁的？

### DISCUSSION THREAD

**Discussion by Xingeqwd** [XTLS/Xray-core#3679] [General]
之前用的是vless+ws+tls+cdn（再之前太古老就不赘述了），不过明显是被破解了，加上便宜线路被针对，老是被封，
所以23年底转到了grpc，实在是慢啊，加上线路不好，延迟超级大，丢包率高，还容易断流，不过所幸是很安全，从没被封过

因为有好几个正经网站在同一台服务器，IP被摸出来就很麻烦，
所以没有CDN太危险了，只得挣扎在CDN的泥潭之中，不然早就去reality（或其他方案）了

之前几个月也有不停的查找关注，不过好像都没有更新更好更强的。。。
到了24年中旬的现在，想想兴许是敝人井底之蛙了，才来请教一下各位大佬。。。

**Comment by RPRX** [comment]
~~愣是没看见 Xray 最近几个版本主推的 SplitHTTP~~

**Reply by Xingeqwd** [reply]
> ~愣是没看见 Xray 最近几个版本主推的 SplitHTTP~

请问大佬，这是论坛吗？或者TG群？能不能给个链接我关注下

**Reply by TwoOnefour** [reply]
@Xingeqwd 

请参见xray-example
- [VLESS-TLS-SplitHTTP-H3](https://github.com/XTLS/Xray-examples/tree/main/VLESS-TLS-SplitHTTP-H3)
- [VLESS-TLS-SplitHTTP-CaddyNginx](https://github.com/XTLS/Xray-examples/tree/main/VLESS-TLS-SplitHTTP-CaddyNginx)

**Reply by Xingeqwd** [reply]
> @Xingeqwd
> 
> 请参见xray-example
> 
> * [VLESS-TLS-SplitHTTP-H3](https://github.com/XTLS/Xray-examples/tree/main/VLESS-TLS-SplitHTTP-H3)
> * [VLESS-TLS-SplitHTTP-CaddyNginx](https://github.com/XTLS/Xray-examples/tree/main/VLESS-TLS-SplitHTTP-CaddyNginx)

谢谢大佬，敝人知道是协议，
我的意思是说，这些最新更新消息的来源是？难道是github的release吗。。。
应该有个TG群或者论坛什么的吧，总感觉鄙人掉队很严重、、、

**Reply by mmmray** [reply]
There is a telegram group and channel (see [README](https://github.com/XTLS/Xray-core?tab=readme-ov-file#telegram)), but the release notes are probably more informative

**Reply by Xingeqwd** [reply]
> There is a telegram group and channel (see [README](https://github.com/XTLS/Xray-core?tab=readme-ov-file#telegram)), but the release notes are probably more informative

thank you sir!
